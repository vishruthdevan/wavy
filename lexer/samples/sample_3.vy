audio1 = load('audio1.wav')
duration = 0.0
for ( duration < 10.0) {
    audio1 = loop(audio1, 1)
    duration = getDuration(audio1)
}

export(audio1, "looped_audio1.wav")