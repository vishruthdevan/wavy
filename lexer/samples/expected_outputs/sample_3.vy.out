<IDENTIFIER, "audio1">
<ASSIGN, "=">
<IDENTIFIER, "load">
<LPR, "(">
<STRING, "audio1.wav">
<RPR, ")">
<IDENTIFIER, "duration">
<ASSIGN, "=">
<FLOAT, "0.0">
<FOR, "for">
<LPR, "(">
<IDENTIFIER, "duration">
<LT, "<">
<FLOAT, "10.0">
<RPR, ")">
<LBRACE, "{">
<IDENTIFIER, "audio1">
<ASSIGN, "=">
<IDENTIFIER, "loop">
<LPR, "(">
<IDENTIFIER, "audio1">
<COMMA, ",">
<INTEGER, "1">
<RPR, ")">
<IDENTIFIER, "duration">
<ASSIGN, "=">
<IDENTIFIER, "getDuration">
<LPR, "(">
<IDENTIFIER, "audio1">
<RPR, ")">
<RBRACE, "}">
<IDENTIFIER, "export">
<LPR, "(">
<IDENTIFIER, "audio1">
<COMMA, ",">
<STRING, "looped_audio1.wav">
<RPR, ")">
<EOF, "">
